internal/controller/driver_controller.go

```diff
@@ -353,7 +353,7 @@ func (r *driverReconcile) reconcileK8sCsiDriver() error {
 			storagev1.FileFSGroupPolicy,
 		),
 	)
-	if nodePlugin := r.driver.Spec.NodePlugin; nodePlugin != nil {
+	if nodePlugin := r.driver.Spec.NodePlugin; nodePlugin != nil && nodePlugin.EnableSeLinuxHostMount != nil {
```

Comment:
Why do we need this check? 
If `nodePlugin.EnableSeLinuxHostMount != nil` then `desiredCsiDriver.Spec.SELinuxMount` need to be nil as well
The check is meaningless

Reply:
If you look at commit description, kube is setting this to `false` and documented as `false` to be default.

Reply:
In that case, the code change does not reflect it and I predict that the next developer will deem it unnecessary and will remove it.

If the intent is to treat a 'nil' value as a false value, or even as the value that is already there (no touch policy) then you need a different change

One of the following should work better:
```
desiredCsiDriver.Spec.SELinuxMount = cmp.Or(nodePlugin.EnableSeLinuxHostMount, ptr.To(false))
```

Or even better:
```
desiredCsiDriver.Spec.SELinuxMount = cmp.Or(nodePlugin.EnableSeLinuxHostMount, desiredCsiDriver.Spec.SELinuxMount)
```

Reply:
done.